docs/static/monitoring/monitoring-output.asciidoc

```diff
@@ -36,6 +38,9 @@ If you are working with a cluster that has {security-features} enabled, extra
 steps are necessary to properly configure Logstash. For more information, see 
 <<configuring-logstash>>. 
 
+
+//TODO:  Verify the following sentence:
+
 IMPORTANT: When discussing security relative to the `elasticsearch` output,
 remember that all users are managed on the production cluster, which is
```

Comment:
This should be the monitoring cluster, and maybe should mention `monitoring.elasticsearch` settings rather than "elasticsearch output" for clarity?